.ci/.jenkins-schema.groovy

```diff
@@ -0,0 +1,12 @@
+---
+agents:
+  - REPO: "apm-agent-dotnet"
```

Comment:
@gregkalapos point me to use https://github.com/elastic/apm-agent-dotnet/tree/master/test/Elastic.Apm.Tests/TestResources/ and the subfolder `json-spec`
```suggestion
  - REPO: "apm-agent-dotnet"
    SPEC_FILEPATH: "test/Elastic.Apm.Tests/TestResources/json-spec"
```